.github/workflows/ci.yml

```diff
@@ -28,7 +28,7 @@ jobs:
         uses: actions/cache@v2.1.4
         with:
           path: node_modules
-          key: ${{ matrix['node-version'] }}-yarn-${{ hashFiles('package-lock.json') }}
```

Comment:
You might want to do this instead to cache and install deps:
https://github.com/zwave-js/node-zwave-js/blob/efe8d4a8c843b1376db1e8ac2873c66476e14a43/.github/workflows/test-and-release.yml#L34-L48

Reply:
essentially I should change the cache key to that one and use `yarn install --prefer-offline --frozen-lockfile`?

Reply:
Getting the cache key and installing the node modules should be changed to the lines I linked. That's what I've found to be best practise for caching yarn installs on CI

Reply:
So all of this:
![grafik](https://user-images.githubusercontent.com/17641229/117264940-076cf880-ae54-11eb-9de6-fa115efeaa2d.png)

And maybe even edit the cache key for your cached build?

Reply:
done